pkgs/build-support/node/build-npm-package/hooks/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, makeSetupHook, nodejs, srcOnly, diffutils, jq, makeWrapper }:
```

For consistency, and also to make the diff easier to read if we change something in the future:
```suggestion
{ lib
, makeSetupHook
, nodejs
, srcOnly
, diffutils
, jq
, makeWrapper
}:
```